packages/flex-plugin-scripts/src/scripts/test.ts

```diff
@@ -1,5 +1,4 @@
 import { logger } from 'flex-dev-utils';
```

Comment:
this file doesn't seem to be doing anything anymore?

Reply:
It will in a subsequent PR for testing.